lib/contentful/bootstrap/commands/create_space.rb

```diff
@@ -48,6 +48,7 @@ def fetch_space
               name: @space
             }
             options[:organization_id] = @token.read_organization_id unless @token.read_organization_id.nil?
+            management_client_init
```

Comment:
`management_client_init` does `@client ||= ...` so basically they are equivalent, but at the point this invocation is made... there were cases (when `trigger_oauth` was false) in which the client would not be present, therefore this invocation is required... i could remove the assignment in the other instance of this invocation... so it's not confusing, but they are completely equivalent